src/cli.ts

```diff
@@ -61,24 +61,31 @@ export const program = new Command();
 
 program
   .command('transpile <files...>')
-  .option('--compile-cairo')
+  .description('Transpile Solidity contracts into Cairo contracts')
+  .option('--compile-cairo', 'Compile the output to bytecode')
   .option('--no-compile-errors')
-  .option('--check-trees')
+  .option('--check-trees', 'Debug: Run sanity checks on all intermediate ASTs')
   // for development mode
   .option('--dev', 'Run AST sanity checks on every pass instead of the final AST only', false)
-  .option('--format-cairo', 'Format cairo output')
-  .option('--highlight <ids...>')
-  .option('--order <passOrder>')
+  .option('--format-cairo', 'Format the cairo output - can be slow on large contracts')
+  .option(
+    '--highlight <ids...>',
+    'Debug: Highlight selected ids in the AST printed by --print-trees',
+  )
+  .option('--order <passOrder>', 'Use a custom set of transpilation passes')
   .option('-o, --output-dir <path>', 'Output directory for transpiled Cairo files.', 'warp_output')
-  .option('-d, --debug-info', 'Include debug information.', false)
-  .option('--print-trees')
-  .option('--no-result')
-  .option('--no-stubs')
-  .option('--no-strict')
+  .option(
+    '-d, --debug-info',
+    'Include debug information in the compiled bytecode produced by --compile-cario',
+    false,
+  )
+  .option('--print-trees', 'Debug: Print all the intermediate ASTs')
+  .option('--no-stubs', 'Debug: Hide the stubs in the intermedidate ASTs when using print-trees')
```

Comment:
```suggestion
  .option('--no-stubs', 'Debug: Hide the stubs in the intermedidate ASTs when using --print-trees')
```

Nit, can you do this for the rest too?